Assets/Mirror/Components/NetworkLobbyPlayer.cs

```diff
@@ -0,0 +1,142 @@
+using System;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Mirror
+{
+    [DisallowMultipleComponent]
+    [AddComponentMenu("Network/NetworkLobbyPlayer")]
+    [HelpURL("https://vis2k.github.io/Mirror/Components/NetworkLobbyPlayer")]
+    public class NetworkLobbyPlayer : NetworkBehaviour
+    {
+        [SerializeField] public bool ShowLobbyGUI = true;
```

Comment:
[serializefield] not needed for public fields

Reply:
Fixed